selftests/checkall

```diff
@@ -5,6 +5,7 @@ run_rc() {
     eval $1
     if [ $? != 0 ]; then
         GR=1
+        [ ! "$SELF_CHECK_CONTINUOUS" ] && exit 1
```

Comment:
I think the presence of `SELF_CHECK_CONTINOUS` would make the `make check` to behave as it is currently, before this patch. The absence of `SELF_CHECK_CONTINOUS` would make the option `failfast=True` to take place in `unittest.TextTestRunner()` and also the `--failfast` option to take place in `AVOCADO_SELF_CHECK` related command.